### PR TITLE
Change email generation function

### PR DESCRIPTION
**Pull request by ChrisLovering**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -6,38 +6,35 @@
...
+    # Choose the case for the username based on the option parameter
     if option == 0:
+        firstname = firstname.upper()
         lastname = lastname.upper()
     elif option == 1:
+        firstname = firstname.lower()
         lastname = lastname.lower()
     else:
+        firstname = firstname.title()
         lastname = lastname.title()
```
Comment (TestedBubble) [inline main.py]: I preferred the variation of having uppercase/title/lowercase for either firstname or lastname as opposed to both of them being the same.

File: main.py [hunk 2]
```diff
@@ -6,38 +6,35 @@
...
-    number = str(random.randint(0, 500))
-
-    finalCombo = {0: firstname + number + lastname,
-                  1: lastname + number + firstname,
-                  2: firstname + lastname + number,
-                  3: lastname + firstname + number}
-
-    return finalCombo[random.randint(0, 3)] + domains[random.randint(0, len(domains)-1)]
+    # Join the username, shuffle it randomly and then return it with the domain appended
+    username = [firstname, lastname, number]
```
Comment (TestedBubble) [inline main.py]: Merge line 21 into here + remove line 20

File: main.py [hunk 3]
```diff
@@ -6,38 +6,35 @@
...
+    random.shuffle(username)
+    return ''.join(username) + domain
```
Comment (TestedBubble) [inline main.py]: Couldn't this just be?:
return username + domain

Comment (ChrisLovering) [reply main.py]: username is a list and domain is a string, so they cannot be concatenated like that

File: main.py (hunk 2 above)
Comment (ChrisLovering) [reply main.py]: Din't quite understand what you want to be done here, line 20&21 are the .upper() calls?

File: main.py (hunk 1 above)
Comment (ChrisLovering) [reply main.py]: Ok, I'll think of another approach for this.

File: main.py (hunk 2 above)
Comment (ChrisLovering) [reply main.py]: Will be done in next commit

File: main.py (hunk 1 above)
Comment (ChrisLovering) [reply main.py]: I have change this to use a new function, see latest commit

**Pull request merged**